test/server/lib/handlebars.spec.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const { expect } = require('chai');
+const Handlebars = require('../../../server/lib/handlebars')();
+
+describe('Handlebars with extended helper', function () {
+	it('Should register an ifEquals helper', function () {
+		expect(Handlebars.helpers).to.haveOwnProperty('ifEquals');
+	});
+
+	it('should render a template using ifEquals helper', function () {
+		const template = Handlebars.compile('{{#ifEquals thing1 "thing1"}}first{{else}}not first{{/ifEquals}} {{#ifEquals thing1 "thing2"}}second{{else}}not second{{/ifEquals}}');
+		const rendered = template({thing1: 'thing1', thing2: 'thing2'});
```

Comment:
![Thing1-and-thing2](https://user-images.githubusercontent.com/10484515/86572610-13093d80-bf6b-11ea-9f45-636361ea8d9a.jpg)

Reply:
Although maybe for clarity the values should be different from the property names, i.e. `{thing1: 'value1', thing2: 'value2'}`

Reply:
fixed :)